sdk/js/src/Request.ts

```diff
@@ -25,27 +25,28 @@ export const EVENT_URL = 'https://events'
 export const CONFIG_PATH = '/v1/sdkConfig'
 export const EVENTS_PATH = '/v1/events'
 
-export const baseRequestHeaders = (environmentKey?: string): AxiosRequestHeaders => {
+export const baseRequestHeaders = (environmentKey?: string, enableCloudData?: boolean): AxiosRequestHeaders => {
     return {
         'Content-Type': 'application/json',
-        ...(environmentKey ? { 'Authorization': environmentKey } : {})
+        ...(environmentKey ? { 'Authorization': environmentKey } : {}),
+        ...(enableCloudData ? {'Enable-Cloud-Entity-Data': enableCloudData} : {})
```

Comment:
checked out the code locally and `enableCloudData` needs to be stringified for typescript not to complain / tests pass